### PR TITLE
telemetry_opt_out: use a different name for the property

### PR DESCRIPTION
**Pull request by goneri**
The `makemigrations` command was confused by the current situation were the column, the Django field and the virtual property were colliding.

As a result it was trying to:
- drop the column
- and recreate it.

This change clarify the model and uses a designed property called `has_telemetry_opt_out` to ready the value.

### CODE REVIEW

File: ansible_ai_connect/organizations/models.py [hunk 1]
```diff
@@ -23,20 +23,19 @@
 
 class Organization(models.Model):
     id = models.IntegerField(primary_key=True)
-    _telemetry_opt_out = models.BooleanField(default=False, db_column="telemetry_opt_out")
+    telemetry_opt_out = models.BooleanField(default=False, db_column="telemetry_opt_out")
+    extra_kwargs = {
+        "telemetry_opt_out": {"write_only": True},
```
Comment (manstis) [inline ansible_ai_connect/organizations/models.py:28]: I don't understand why you're adding the `extra_kwargs` for `write_only: True`. Can you explain?

**Review by manstis — COMMENTED** [review]
LGTM 👍 

Just a question before approving.

File: ansible_ai_connect/organizations/models.py (hunk 1 above)
Comment (goneri) [reply ansible_ai_connect/organizations/models.py]: It was to turn the flag read-only as a safeguard and be sure we only use the has_ property instead.

Comment (goneri) [reply ansible_ai_connect/organizations/models.py]: https://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields

Comment (goneri) [reply ansible_ai_connect/organizations/models.py]: Actually, ignore what I just said. I was missing the serializer and the model...

File: ansible_ai_connect/organizations/migrations/0003_alter_organization_telemetry_opt_out.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.11 on 2024-07-02 17:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (manstis) [inline ansible_ai_connect/organizations/migrations/0003_alter_organization_telemetry_opt_out.py:6]: Do we need to worry about data migration?

Will this rename a database column or create a new one and delete the old?

**Review by manstis — COMMENTED** [review]
Sorry, your mention of the model and serialiser made me think.

File: ansible_ai_connect/organizations/migrations/0003_alter_organization_telemetry_opt_out.py (hunk 2 above)
Comment (goneri) [reply ansible_ai_connect/organizations/migrations/0003_alter_organization_telemetry_opt_out.py:6]: The migration script doesn't change anything. This is already the name in the DB. But the problem was that `makemigrations` has never been called after the previous model changes.

**Review by manstis — APPROVED** [review]
Thanks @goneri LGTM 👍

**Pull request merged**